src/Rel8/Schema/Column.hs

```diff
@@ -90,7 +89,6 @@ type Column context a =
 type HEither :: K.Context -> Type -> Type -> Type
 type family HEither context where
   HEither Structure = Shape2 'Either
-  HEither Aggregate = EitherTable
```

Comment:
It _might_ be possible to get this working, but I haven't explored it. I'm still not very interested in this API, but if you want to get it working I'll accept whatever is needed to make this work.